tensilelite/Tensile/Source/lib/source/ContractionProblem.cpp

```diff
@@ -1065,6 +1066,42 @@ namespace Tensile
             TENSILE_ASSERT_EXC(dUse == 1);
     }
 
+    void ContractionProblem::calcArithmeticIntensity()
+    {
```

Comment:
I think this function is going to have to be part of the ContractionProblemGemm class in hipBLASLt. These classes got a bit re-arranged in this repo to allow grouped gemm and the size info used here is now in the subclass.